docs/shell-tips.md

```diff
@@ -158,16 +191,146 @@ and hit <kbd>ENTER</kbd>/<kbd>RETURN</kbd> at the end.
 
 If you think about it, what is happening here is actually another case of
 character escaping. The newline character (the one that appears when you hit
-<kbd>ENTER</kbd>) usually means "read this command". However, here we want to literally
-have the newline character, and thus the `\<newline>`.
+<kbd>ENTER</kbd>) usually means "read this command". However, here we want to
+literally have the newline character, and thus the `\<newline>`.
 
 The newline character is invisible (we only see a line break), but it's still
 there!
 
+## Arguments
+
+Most commands need additional data to work, like a path or a file. That extra
+information is called an **argument**, and it's specified after the name of the
+command, like this:
+
+```
+$ cd /home/john/notes
+```
+
+Here, the command is `cd`, and the first (and only) argument is
+`/home/john/notes`:
+
+- `cd` - *command*: app that changes your current directory.
+
+- `/home/john/notes` - *argument*: the directory where you want to go.
+
+In each command the arguments are specified in different ways, and have
+different meanings.
+
+Sometimes, a command can accept multiple arguments. In those cases, dashes are
+used to indicate the different arguments. Here's another example:
```

Comment:
By hash style argument I meant like you specify the type of the argument and the value with it, e.g: `--arg=val`. But yeah you don't need to include that,

I think just saying that a command can also accept arguments using dash is fine.